cl/search/models.py

```diff
@@ -3320,6 +3320,14 @@ class Opinion(AbstractDateTimeModel):
             "sha1",
         ]
     )
+    order = models.IntegerField(null=True, blank=True)
```

Comment:
I'm just passing through here, but I'm not sure that `order` is the right name for this. it has two problems:

1. It could be interpreted as a noun because courts issue orders, so at a glance you might think it's a boolean that represents whether the opinion row is an order. Hrm.

2. It's quite generic, and intersects in a lot of other things in the DB. For example, this is a bit weird: `Opinion.objects.order_by('order')`.

I _think_  this field is more like `ordering_key` or even `cluster_order_key`. I don't love those either. Other ideas?

Reply:
mmm maybe `position` or `spot`?

Reply:
`spot` feels wrong to me. We have a `position` field on the courts table that deals with sorting, but it also can have two meanings. It can also mean the role that somebody has in an organization.

I think my fave remains `ordering_key`. 

@flooie do you have any thoughts?

Reply:
How about something like `publication_order`?